easybuild/easyconfigs/w/WPS/WPS-4.0.1-intel-2018b-dmpar.eb

```diff
@@ -0,0 +1,36 @@
+name = 'WPS'
+version = '4.0.1'
+
+homepage = 'http://www.wrf-model.org'
+description = """WRF Preprocessing System (WPS) for WRF. The Weather Research and Forecasting (WRF) Model is 
+ a next-generation mesoscale numerical weather prediction system designed to serve both operational 
+ forecasting and atmospheric research needs."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/wrf-model/WPS/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'WPS-%(version)s_netCDF-Fortran_seperate_path.patch',
+    'WPS-%(version)s_find-wrfdir.patch',
+]
+checksums = [
+    '1379cee8029818e7a69376a7c425d1a00801284182dd0f56a58a753703d5fb2b',  # v4.0.1.tar.gz
+    'ca8e931847658b0fe24283dc16a87479c8093571f4bae9a04fa1ab6cad49bcb1',  # WPS-4.0.1_netCDF-Fortran_seperate_path.patch
+    '279fc79909a3eb0370fea0bdf5690e0d5bfb9c5d04350f10ae0bf48680238111',  # WPS-4.0.1_find-wrfdir.patch
+]
+
+buildtype = "dmpar"
+versionsuffix = '-%s' % buildtype
+
+dependencies = [
+    ('WRF', version, '%s' % versionsuffix),
```

Comment:
@andreas-h This is a bit silly, the `'%s'` is pointless, should be:

```python
('WRF', version, versionsuffix),

Reply:
agreed :-)